app/src/main/java/com/study/dongamboard/api/Utils.kt

```diff
@@ -0,0 +1,51 @@
+package com.study.dongamboard.api
+
+import android.content.Context
+import android.util.Log
+import com.skydoves.sandwich.StatusCode
+import com.study.dongamboard.type.ResponseStatus
+
+class Utils(context: Context) {
+    private val TAG = "TEST"
+    private val DEBUG = true
+
+    fun logD(message: String) {
+        if (DEBUG) {
+            Log.d(TAG, message)
+        }
+    }
+
+    fun logD(statusCode: StatusCode) {
+        var successMsg = ""
+        when (statusCode.code) {
+            ResponseStatus.SUCCESS.code -> successMsg = ResponseStatus.SUCCESS.message
+            ResponseStatus.CREATED.code -> successMsg = ResponseStatus.CREATED.message
+        }
+        if (DEBUG) {
+            Log.d(TAG, "$statusCode $successMsg")
+        }
+    }
+
+    fun logE(message: String) {
+        if (DEBUG) {
+            Log.e(TAG, message)
+        }
+    }
+
+    fun logE(statusCode: StatusCode): String {
+        var errorMsg = ""
```

Comment:
아래와 같이 바꿔도 괜찮을 것 같아요.
```suggestion
        var errorMsg: String
```

Reply:
저도 신경쓰였는데, 아래에서 사용하니 미리 선언하라고 하더군요.. 다시 확인해보겠습니다!